--- lynx2-8-7/src/HTML.c~	2009-06-23 23:53:58.000000000 +0000
+++ lynx2-8-7/src/HTML.c	2015-01-01 18:21:04.617496824 +0000
@@ -2716,13 +2716,13 @@
 		    }
 		}
 		if (seqtype == 'A') {
-		    sprintf(number_string, LYUppercaseA_OL_String(seqnum));
+		    sprintf(number_string, "%s", LYUppercaseA_OL_String(seqnum));
 		} else if (seqtype == 'a') {
-		    sprintf(number_string, LYLowercaseA_OL_String(seqnum));
+		    sprintf(number_string, "%s", LYLowercaseA_OL_String(seqnum));
 		} else if (seqtype == 'I') {
-		    sprintf(number_string, LYUppercaseI_OL_String(seqnum));
+		    sprintf(number_string, "%s", LYUppercaseI_OL_String(seqnum));
 		} else if (seqtype == 'i') {
-		    sprintf(number_string, LYLowercaseI_OL_String(seqnum));
+		    sprintf(number_string, "%s", LYLowercaseI_OL_String(seqnum));
 		} else {
 		    sprintf(number_string, "%2d.", seqnum);
 		}
