diff -burN lynx2-8-5.orig/aclocal.m4 lynx2-8-5/aclocal.m4
--- lynx2-8-5.orig/aclocal.m4	2004-02-14 22:10:58.336429456 +0100
+++ lynx2-8-5/aclocal.m4	2004-02-14 22:14:16.154356560 +0100
@@ -24,7 +24,6 @@
    AC_REQUIRE([AC_HEADER_STDC])dnl
    AC_REQUIRE([AC_C_CONST])dnl
    AC_REQUIRE([AC_C_INLINE])dnl
-   AC_REQUIRE([AC_TYPE_OFF_T])dnl
    AC_REQUIRE([AC_TYPE_SIZE_T])dnl
    AC_REQUIRE([AC_FUNC_ALLOCA])dnl
    AC_REQUIRE([AC_FUNC_MMAP])dnl
@@ -2226,9 +2226,7 @@
 dnl CF_HELP_MESSAGE version: 3 updated: 1998/01/14 10:56:23
 dnl ---------------
 dnl Insert text into the help-message, for readability, from AC_ARG_WITH.
-AC_DEFUN([CF_HELP_MESSAGE],
-[AC_DIVERT_HELP([$1])dnl
-])dnl
+AC_DEFUN([CF_HELP_MESSAGE],[])
 dnl ---------------------------------------------------------------------------
 dnl CF_INET_ADDR version: 4 updated: 1999/03/30 12:24:31
 dnl ------------
--- lynx2-8-6/configure.in.orig	2006-10-11 01:39:50.000000000 +0200
+++ lynx2-8-6/configure.in	2007-02-25 20:19:36.781239762 +0100
@@ -36,7 +36,6 @@
 # run to the next, as well as with tests that are order-dependent.
 rm -f config.cache
 
-AC_PREREQ(2.13.20020210)
 AC_INIT(userdefs.h)
 
 # autoconf 2.5x defaults to no cache file; we need the cache file's information
@@ -101,9 +100,6 @@
 dnl Debug/development/test
 dnl --------------------------------------------------------------------------
 
-CF_HELP_MESSAGE(
-Development Options:)
-
 CF_DISABLE_ECHO
 
 # special case for WWW/*
@@ -335,9 +331,6 @@
 dnl --------------------------------------------------------------------------
 dnl --------------------------------------------------------------------------
 
-CF_HELP_MESSAGE(
-Basic Configuration Options:)
-
 CF_LARGEFILE
 
 dnl internationalization macros
@@ -658,9 +651,6 @@
 dnl Experimental options that don't need to test system features
 dnl --------------------------------------------------------------------------
 
-CF_HELP_MESSAGE(
-Experimental Options:)
-
 AC_MSG_CHECKING(if bibp: URLs should be supported)
 CF_ARG_DISABLE(bibp-urls,
 [  --disable-bibp-urls     disable support for bibp: URLs],
@@ -849,9 +839,6 @@
 dnl Miscellaneous options that don't need to test system features
 dnl --------------------------------------------------------------------------
 
-CF_HELP_MESSAGE(
-Miscellaneous Options:)
-
 AC_MSG_CHECKING(if alternative line-edit bindings should be used)
 CF_ARG_ENABLE(alt-bindings,
 [  --disable-alt-bindings  disable alternative line-edit bindings],
@@ -1095,9 +1082,6 @@
 dnl DirEd (directory-editor) options
 dnl --------------------------------------------------------------------------
 
-CF_HELP_MESSAGE(
-Directory Editor Options:)
-
 # All DirEd functions that were enabled on compilation can be disabled
 # or modified at run time via DIRED_MENU symbols in lynx.cfg.
 AC_MSG_CHECKING(if directory-editor code should be used)
@@ -1218,7 +1202,6 @@
 AC_SUBST(COMPRESS_EXT)
 
 if test $cf_cv_screen = pdcurses ; then
-	CF_HELP_MESSAGE(Special Libraries for PDCurses X11:)
 	CF_PDCURSES_X11
 else
 	CF_TERMIO_AND_CURSES(LYCurses.h)
