diff -Nur lynx2-8-4.orig/configure.in lynx2-8-4/configure.in
--- lynx2-8-4.orig/configure.in	Sun Jun 10 16:24:29 2001
+++ lynx2-8-4/configure.in	Sun Jun 10 16:16:44 2001
@@ -434,7 +434,7 @@
 cf_libdir=`echo $cf_libdir | sed -e s@'^NONE/'@$prefix/@ -e s@'^NONE/'@$ac_default_prefix/@`
 
 AC_MSG_CHECKING(for location of config-file)
-LYNX_CFG_FILE=$cf_libdir/lynx.cfg
+LYNX_CFG_FILE=$sysconfdir/lynx.cfg
 AC_DEFINE_UNQUOTED(LYNX_CFG_FILE,"$LYNX_CFG_FILE")
 AC_MSG_RESULT($LYNX_CFG_FILE)
 
@@ -696,7 +696,7 @@
 	AC_MSG_RESULT(yes)
 
 	AC_MSG_CHECKING(for location of style-sheet file)
-	LYNX_LSS_FILE=$cf_libdir/lynx.lss
+	LYNX_LSS_FILE=$datadir/lynx/lynx.lss
 	AC_DEFINE_UNQUOTED(LYNX_LSS_FILE,"$LYNX_LSS_FILE")
 	AC_MSG_RESULT($LYNX_LSS_FILE)
 	INSTALL_LSS=install-lss
diff -Nur lynx2-8-5.orig/lynx.cfg lynx2-8-5/lynx.cfg
--- lynx2-8-5.orig/lynx.cfg	Sun Jun 10 16:24:29 2001
+++ lynx2-8-5/lynx.cfg	Sun Jun 10 16:16:44 2001
@@ -1,5 +1,5 @@
 # lynx.cfg file.
-# The default placement for this file is /usr/local/lib/lynx.cfg (Unix)
+# The default placement for this file is /etc/lynx.cfg (Unix)
 #                                     or Lynx_Dir:lynx.cfg (VMS)
 #
 # $Format: "#PRCS LYNX_VERSION \"$ProjectVersion$\""$
@@ -11,7 +11,7 @@
 # Definition pairs are of the form  VARIABLE:DEFINITION
 # NO spaces are allowed between the pair items.
 #
-#  If you do not have write access to /usr/local/lib you may change
+#  If you do not have write access to /etc you may change
 #  the default location of this file in the userdefs.h file and recompile,
 #  or specify its location on the command line with the "-cfg"
 #  command line option.
@@ -42,8 +42,8 @@
 #
 # Then in ~/lynx.cfg:
 #
-#   INCLUDE:/usr/local/lib/lynx.cfg
-#           ^^^^^^^^^^^^^^^^^^^^^^^ or whatever is appropriate on your system
+#   INCLUDE:/etc/lynx.cfg
+#           ^^^^^^^^^^^^^ or whatever is appropriate on your system
 # and now your own tweaks.
 #
 # Starting with Lynx 2.8.2, the INCLUDE facility is yet more powerful.  You can
@@ -88,7 +88,8 @@
 # Normally we expect you will connect to a remote site, e.g., the Lynx starting
 # site:
 #STARTFILE:http://lynx.isc.org/
-STARTFILE:file://localhost/usr/share/doc/HTML/index.html
+#STARTFILE:file://localhost/usr/share/doc/HTML/index.html
+STARTFILE:file://localhost/~/
 #
 # As an alternative, you may want to use a local URL.  A good choice for this is
 # the user's home directory:
@@ -112,9 +113,8 @@
 #   This should be changed to the local path.
 #
 #HELPFILE:http://www.subir.com/lynx/lynx_help/lynx_help_main.html
-.ex
-HELPFILE:file://localhost/usr/share/lynx/help/lynx_help_main.html
-
+# default in PLD is:
+#HELPFILE:file://localhost/usr/share/lynx/help/lynx_help_main.html.gz
 
 .h2 DEFAULT_INDEX_FILE
 # DEFAULT_INDEX_FILE is the default file retrieved when the
diff -Nur lynx2-8-5.orig/makefile.in lynx2-8-5/makefile.in
--- lynx2-8-5.orig/makefile.in	Mon Apr  2 02:51:46 2001
+++ lynx2-8-5/makefile.in	Sun Jun 10 16:29:52 2001
@@ -48,15 +48,15 @@
 MANDIR= $(DESTDIR)$(mandir)
 
 ## Where you want the lynx.cfg file installed
-libdir= @libdir@
+libdir= @sysconfdir@
 LIBDIR= $(DESTDIR)$(libdir)
 
 ## Where you want the doc-files installed
-docdir= @libdir@/lynx_doc
+docdir= @datadir@/lynx_doc
 DOCDIR= $(DESTDIR)$(docdir)
 
 ## Where you want the help-files installed
-helpdir= @libdir@/lynx_help
+helpdir= @datadir@/lynx/help
 HELPDIR= $(DESTDIR)$(helpdir)
 
 ##set the relative location of the WWW library Implementation directory,
@@ -326,7 +326,7 @@
 	@-rm -f help_files.tmp
 
 install-help : help_files.sed $(HELPDIR)
-	- (cd $(HELPDIR) && WD=`pwd` && HEAD=`echo $$WD|sed -e 's!/lynx_help$$!!'` && test $$WD != $$HEAD && rm -fr *)
+	- (cd $(HELPDIR) && WD=`pwd` && HEAD=`echo $$WD|sed -e 's!/lynx/help$$!!'` && test $$WD != $$HEAD && rm -fr *)
 	test -d $(HELPDIR)/keystrokes || mkdir $(HELPDIR)/keystrokes
 @LYNXCFG_MAKE@	@echo 'Making htmlized lynx.cfg'
 @LYNXCFG_MAKE@	cd src && make LYReadCFG.i
@@ -438,12 +438,12 @@
 
 uninstall \
 uninstall-help ::
-	-test -d $(HELPDIR) && WD=`cd $(HELPDIR) && pwd` && HEAD=`echo $$WD|sed -e 's!/lynx_help$$!!'` && test $$WD != $$HEAD && rm -rf $(HELPDIR)
+	-test -d $(HELPDIR) && WD=`cd $(HELPDIR) && pwd` && HEAD=`echo $$WD|sed -e 's!/lynx/help$$!!'` && test $$WD != $$HEAD && rm -rf $(HELPDIR)
 
 uninstall \
 uninstall-doc ::
 	-test -d $(DOCDIR) && WD=`cd $(DOCDIR) && pwd` && HEAD=`echo $$WD|sed -e 's!/lynx_doc$$!!'` && test $$WD != $$HEAD && rm -rf $(DOCDIR)
-	-test -d $(HELPDIR) && WD=`cd $(HELPDIR) && pwd` && HEAD=`echo $$WD|sed -e 's!/lynx_help$$!!'` && test $$WD != $$HEAD && cd $(HELPDIR) && rm -f COPYING COPYHEADER
+	-test -d $(HELPDIR) && WD=`cd $(HELPDIR) && pwd` && HEAD=`echo $$WD|sed -e 's!/lynx/help$$!!'` && test $$WD != $$HEAD && cd $(HELPDIR) && rm -f COPYING COPYHEADER
 
 $(BINDIR) $(MANDIR) $(LIBDIR) $(HELPDIR) $(DOCDIR) :
 	$(srcdir)/mkdirs.sh $@
diff -Nur lynx2-8-4.orig/userdefs.h lynx2-8-4/userdefs.h
--- lynx2-8-4.orig/userdefs.h	Sun Jun 10 16:24:29 2001
+++ lynx2-8-4/userdefs.h	Sun Jun 10 16:16:44 2001
@@ -93,7 +93,7 @@
  * mailcap files (see the examples in the samples directory).
  */
 #ifndef LYNX_CFG_FILE
-#define LYNX_CFG_FILE "/usr/share/lynx/lynx.cfg"
+#define LYNX_CFG_FILE "/etc/lynx.cfg"
 #endif /* LYNX_CFG_FILE */
 
 /**************************
@@ -277,7 +277,7 @@
 #ifdef DOSPATH
 #define LYNX_CFG_FILE "./lynx.cfg"
 #else
-#define LYNX_CFG_FILE "/usr/share/lynx/lynx.cfg"
+#define LYNX_CFG_FILE "/etc/lynx.cfg"
 #endif /* DOSPATH */
 #endif /* LYNX_CFG_FILE */
 #endif /* HAVE_CONFIG_H */
@@ -478,7 +478,7 @@
  * http://www.subir.com/lynx/lynx_help/lynx_help_main.html
  *   This should be changed here or in lynx.cfg to the local path.
  */
-#define HELPFILE "http://www.subir.com/lynx/lynx_help/lynx_help_main.html"
+#define HELPFILE "file://localhost/usr/share/lynx/help/lynx_help_main.html.gz"
 /* #define HELPFILE "file://localhost/PATH_TO/lynx_help/lynx_help_main.html" */
 
 /*****************************
