--- lynx2.9.2/lynx.cfg.orig	2024-10-26 16:03:27.748973156 +0200
+++ lynx2.9.2/lynx.cfg	2024-10-26 16:06:56.331176502 +0200
@@ -1,6 +1,6 @@
 # $LynxId: lynx.cfg,v 1.358 2024/05/31 22:09:16 tom Exp $
 # lynx.cfg file.
-# The default placement for this file is /usr/local/lib/lynx.cfg (Unix)
+# The default placement for this file is /etc/lynx.cfg (Unix)
 #                                     or Lynx_Dir:lynx.cfg (VMS)
 #
 # LYNX_VERSION "2.9.2"
@@ -10,7 +10,7 @@
 #	VARIABLE:DEFINITION
 # NO spaces are allowed around the colon ":" between the pair items.
 #
-#  If you do not have write access to /usr/local/lib you may change
+#  If you do not have write access to /etc you may change
 #  the default location of this file in the userdefs.h file and recompile,
 #  or specify its location on the command line with the "-cfg"
 #  command line option.
@@ -55,7 +55,7 @@
 #
 # Then in ~/lynx.cfg:
 #
-#   INCLUDE:/usr/local/lib/lynx.cfg
+#   INCLUDE:/etc/lynx.cfg
 #           ^^^^^^^^^^^^^^^^^^^^^^^ or whatever is appropriate on your system
 # and now your own tweaks.  If you omit the directory name, e.g.,
 #
--- lynx2-8-7/makefile.in~	2010-06-21 11:27:35.000000000 +0200
+++ lynx2-8-7/makefile.in	2010-06-21 14:04:23.622318548 +0200
@@ -66,7 +66,7 @@
 DOCDIR    = $(DESTDIR)$(docdir)
 
 ## Where you want the help-files installed
-helpdir   = @datadir@/lynx_help
+helpdir   = @datadir@/lynx/help
 HELPDIR   = $(DESTDIR)$(helpdir)
 
 ##set the relative location of the WWW library Implementation directory,
--- lynx2-8-7/userdefs.h~	2010-06-21 14:03:36.879822401 +0200
+++ lynx2-8-7/userdefs.h	2010-06-21 14:05:52.335642938 +0200
@@ -105,7 +105,7 @@
  * mailcap files (see the examples in the samples directory).
  */
 #ifndef LYNX_CFG_FILE
-#define LYNX_CFG_FILE "/usr/share/lynx/lynx.cfg"
+#define LYNX_CFG_FILE "/etc/lynx.cfg"
 #endif /* LYNX_CFG_FILE */
 
 #ifndef LYNX_CFG_PATH
@@ -510,8 +510,9 @@
  * The definition here can be overridden at run time by defining a
  * "LYNX_HELPFILE" environment variable.
  */
-#define HELPFILE "https://lynx.invisible-island.net/lynx_help/lynx_help_main.html"
+/* ##define HELPFILE "https://lynx.invisible-island.net/lynx_help/lynx_help_main.html" */
 /* #define HELPFILE "file://localhost/PATH_TO/lynx_help/lynx_help_main.html" */
+#define HELPFILE "file://localhost/usr/share/lynx/help/lynx_help_main.html.gz"
 
 /*****************************
  * DEFAULT_INDEX_FILE is the default file retrieved when the
