diff -Nru lynx2-8-3.orig/src/LYMail.c lynx2-8-3/src/LYMail.c
--- lynx2-8-3.orig/src/LYMail.c	Fri May  5 00:43:18 2000
+++ lynx2-8-3/src/LYMail.c	Fri May  5 00:45:15 2000
@@ -392,7 +392,7 @@
      *	entry, if permitted. - FM
      */
     if (!LYNoCc) {
-	sprintf(self, "%.79s", (personal_mail_address ?
+	snprintf(self, 80, "%.79s", (personal_mail_address ?
 				personal_mail_address : ""));
 	self[79] = '\0';
 	_statusline("Cc: ");
@@ -1557,7 +1557,7 @@
     addstr(CTRL_U_TO_ERASE);
     addstr(SUBJECT_PROMPT);
     /* Add the default subject. */
-    sprintf(user_input, "%.70s%.63s",
+    snprintf(user_input, sizeof(user_input), "%.70s%.63s",
 			(subject[0] != '\0') ?
 				     subject :
 		    ((filename && *filename) ?
