--- lynx2-8-4/WWW/Library/Implementation/www_tcp.h.orig	Wed Oct 25 18:35:28 2000
+++ lynx2-8-4/WWW/Library/Implementation/www_tcp.h	Thu Apr  5 13:42:01 2001
@@ -826,6 +826,10 @@
 #ifdef SIN6_LEN
 #define SOCKADDR_LEN(soc_address) ((struct sockaddr *)&soc_address)->sa_len
 #else
+#ifndef SA_LEN
+#define SA_LEN(x) (((x)->sa_family == AF_INET6)?sizeof(struct sockaddr_in6): \
+	(((x)->sa_family == AF_INET)?sizeof(struct sockaddr_in):sizeof(struct sockaddr)))
+#endif
 #define SOCKADDR_LEN(soc_address) SA_LEN((struct sockaddr *)&soc_address)
 #endif /* SIN6_LEN */
 #else
