--- lynx2-8-7/lynx.cfg~	2010-06-21 11:27:35.000000000 +0200
+++ lynx2-8-7/lynx.cfg	2010-06-21 13:35:12.625827117 +0200
@@ -108,7 +108,7 @@
 #
 # Normally we expect you will connect to a remote site, e.g., the Lynx starting
 # site:
-STARTFILE:https://lynx.invisible-island.net/
+STARTFILE:https://localhost/~/
 #
 # As an alternative, you may want to use a local URL.  A good choice for this is
 # the user's home directory:
@@ -132,9 +132,10 @@
 # This definition will be overridden if the "LYNX_HELPFILE" environment
 # variable has been set.
 #
-HELPFILE:https://lynx.invisible-island.net/lynx_help/lynx_help_main.html
+#HELPFILE:https://lynx.invisible-island.net/lynx_help/lynx_help_main.html
 .ex
 #HELPFILE:file://localhost/PATH_TO/lynx_help/lynx_help_main.html
+HELPFILE:file://localhost/usr/share/lynx/help/lynx_help_main.html.gz
 
 .h2 DEFAULT_INDEX_FILE
 # DEFAULT_INDEX_FILE is the default file retrieved when the
@@ -553,7 +554,7 @@
 # Older browser versions will fail so keep NCR_IN_BOOKMARKS:FALSE if you
 # plan to use them.
 #
-#NCR_IN_BOOKMARKS:FALSE
+NCR_IN_BOOKMARKS:TRUE
 
 .h2 FORCE_8BIT_TOUPPER
 # FORCE_8BIT_TOUPPER overrides locale settings and uses internal 8-bit
@@ -2998,9 +2998,18 @@ MINIMAL_COMMENTS:TRUE
 #COLOR:4:magenta:white
 #COLOR:5:blue:white
 #COLOR:6:red:white
-COLOR:6:brightred:black
+#COLOR:6:brightred:black
 #COLOR:7:magenta:cyan
 
+COLOR:0:black:lightgray
+COLOR:1:blue:lightgray
+COLOR:2:yellow:blue
+COLOR:3:black:lightgray
+COLOR:4:black:lightgray
+COLOR:5:blue:lightgray
+COLOR:6:white:blue
+COLOR:7:lightgray:black
+
 .h2 COLOR_STYLE
 # Also known as "lss" (lynx style-sheet), the color-style file assigns color
 # combination to tags and combinations of tags.  Normally a non-empty value
