--- lynx2-8-6/WWW/Library/Implementation/HTFile.c.orig	2006-05-30 00:44:54.000000000 +0200
+++ lynx2-8-6/WWW/Library/Implementation/HTFile.c	2007-02-25 21:40:38.930317847 +0100
@@ -2752,9 +2752,16 @@
 #endif
 	if (HTStat(localname, &dir_info) == -1)		/* get file information */
 	{
-	    /* if can't read file information */
+	    char *tmp_localname = (char *)malloc(strlen(localname) + 4);
 	    CTRACE((tfp, "HTLoadFile: can't stat %s\n", localname));
 
+	    if (tmp_localname) {
+		stpcpy( stpcpy( tmp_localname, localname), ".gz" );
+		FREE(localname);
+		localname = tmp_localname;
+	    }
+	    /* if can't read file information */
+
 	} else {		/* Stat was OK */
 
 	    if (S_ISDIR(dir_info.st_mode)) {
