--- lynx2-8-5/configure.in.orig	Sun Oct  7 00:02:28 2001
+++ lynx2-8-5/configure.in	Tue Oct  9 05:17:14 2001
@@ -3,7 +3,6 @@
 dnl by T.E.Dickey <dickey@invisible-island.net>
 dnl and Jim Spath <jspath@mail.bcpl.lib.md.us>
 
-AC_PREREQ(2.12.971222)
 AC_INIT(userdefs.h)
 rm -f config.cache; touch config.cache
 CONFIG_H=lynx_cfg.h
@@ -51,9 +50,6 @@
 dnl Debug/development/test
 dnl --------------------------------------------------------------------------
 
-CF_HELP_MESSAGE(
-Development Options:)
-
 CF_DISABLE_ECHO
 
 # special case for WWW/*
@@ -268,9 +264,6 @@
 dnl --------------------------------------------------------------------------
 dnl --------------------------------------------------------------------------
 
-CF_HELP_MESSAGE(
-Basic Configuration Options:)
-
 dnl internationalization macros
 CF_BUNDLED_INTL(makefile)
 
@@ -625,9 +618,6 @@
 dnl Experimental options that don't need to test system features
 dnl --------------------------------------------------------------------------
 
-CF_HELP_MESSAGE(
-Experimental Options:)
-
 AC_MSG_CHECKING(if bibp: URLs should be supported)
 CF_ARG_DISABLE(bibp-urls,
 [  --disable-bibp-urls     disable support for bibp: URLs],
@@ -841,9 +831,6 @@
 dnl Miscellaneous options that don't need to test system features
 dnl --------------------------------------------------------------------------
 
-CF_HELP_MESSAGE(
-Miscellaneous Options:)
-
 AC_MSG_CHECKING(if alternative line-edit bindings should be used)
 CF_ARG_ENABLE(alt-bindings,
 [  --disable-alt-bindings  disable alternative line-edit bindings],
@@ -1040,9 +1027,6 @@
 dnl DirEd (directory-editor) options
 dnl --------------------------------------------------------------------------
 
-CF_HELP_MESSAGE(
-Directory Editor Options:)
-
 # All DirEd functions that were enabled on compilation can be disabled
 # or modified at run time via DIRED_MENU symbols in lynx.cfg.
 AC_MSG_CHECKING(if directory-editor code should be used)
@@ -1114,7 +1098,6 @@
 dnl Test features that rely on configure-definitions
 dnl --------------------------------------------------------------------------
 if test $cf_cv_screen = pdcurses ; then
-	CF_HELP_MESSAGE(Special Libraries for PDCurses X11:)
 	CF_PDCURSES_X11
 else
 	CF_TERMIO_AND_CURSES(LYCurses.h)
