--- lynx2-8-5/configure.in.orig	Sat Jul  6 18:54:26 2002
+++ lynx2-8-5/configure.in	Sat Jul  6 19:00:22 2002
@@ -863,7 +863,7 @@
 AC_MSG_RESULT($use_externs)
 if test $use_externs != "no" ; then
 	AC_DEFINE(USE_EXTERNALS)
-	LIBOBJS="$LIBOBJS LYExtern.o"
+	AC_LIBOBJ([LYExtern])
 fi
 
 AC_MSG_CHECKING(if you want to use setfont support)
@@ -1037,7 +1037,7 @@
 AC_MSG_RESULT($use_dired)
 
 if test ".$use_dired" != ".no" ; then
-	LIBOBJS="$LIBOBJS LYLocal.o"
+	AC_LIBOBJ([LYLocal])
 	AC_DEFINE(DIRED_SUPPORT)
 
 	AC_MSG_CHECKING(if you wish to allow extracting from archives via DirEd)
