diff -ru lynx2-8-2.old/userdefs.h lynx2-8-2/userdefs.h
--- lynx2-8-2.old/userdefs.h	Tue May 11 22:20:02 1999
+++ lynx2-8-2/userdefs.h	Tue May 11 22:25:46 1999
@@ -93,7 +93,7 @@
  * mailcap files (see the examples in the samples directory).
  */
 #ifndef LYNX_CFG_FILE
-#define LYNX_CFG_FILE "Lynx_Dir:lynx.cfg"
+#define LYNX_CFG_FILE "/usr/share/lynx/lynx.cfg"
 #endif /* LYNX_CFG_FILE */
 
 /**************************
@@ -103,8 +103,8 @@
  * Mappings in these global and personal files override any SUFFIX
  * definitions in lynx.cfg and built-in defaults from src/HTInit.c.
  */
-#define GLOBAL_EXTENSION_MAP "Lynx_Dir:mime.types"
-#define PERSONAL_EXTENSION_MAP "mime.types"
+#define GLOBAL_EXTENSION_MAP "/etc/mime.types"
+#define PERSONAL_EXTENSION_MAP ".mime.types"
 
 /**************************
  * The MAILCAP file allows you to map file MIME types to
@@ -113,7 +113,7 @@
  * Mappings in these global and personal files override any VIEWER
  * definitions in lynx.cfg and built-in defaults from src/HTInit.c.
  */
-#define GLOBAL_MAILCAP "Lynx_Dir:mailcap"
+#define GLOBAL_MAILCAP "/etc/mailcap"
 #define PERSONAL_MAILCAP ".mailcap"
 
 /**************************
@@ -276,7 +276,7 @@
 #ifdef DOSPATH
 #define LYNX_CFG_FILE "./lynx.cfg"
 #else
-#define LYNX_CFG_FILE "/usr/local/lib/lynx.cfg"
+#define LYNX_CFG_FILE "/usr/share/lynx/lynx.cfg"
 #endif /* DOSPATH */
 #endif /* LYNX_CFG_FILE */
 #endif /* HAVE_CONFIG_H */
@@ -288,7 +288,7 @@
  * Mappings in these global and personal files override any SUFFIX
  * definitions in lynx.cfg and built-in defaults from src/HTInit.c.
  */
-#define GLOBAL_EXTENSION_MAP "/usr/local/lib/mosaic/mime.types"
+#define GLOBAL_EXTENSION_MAP "/etc/mime.types"
 #define PERSONAL_EXTENSION_MAP ".mime.types"
 
 /**************************
@@ -298,7 +298,7 @@
  * Mappings in these global and personal files override any VIEWER
  * definitions in lynx.cfg and built-in defaults from src/HTInit.c.
  */
-#define GLOBAL_MAILCAP "/usr/local/lib/mosaic/mailcap"
+#define GLOBAL_MAILCAP "/etc/mailcap"
 #define PERSONAL_MAILCAP ".mailcap"
 
 /**************************
@@ -410,7 +410,7 @@
  * -lss command line switch will override these definitions.
  */
 #ifndef LYNX_LSS_FILE
-#define LYNX_LSS_FILE "/usr/local/lib/lynx.lss"
+#define LYNX_LSS_FILE "/usr/share/lynx/lynx.lss"
 #endif /* LYNX_LSS_FILE */
 
 #endif /* VMS OR UNIX */
@@ -1030,7 +1030,7 @@
  * lynxcgi:advice  as a G)oto URL for more information and links to the
  * OSU server distribution.
  */
-/* #define LYNXCGI_LINKS */
+#define LYNXCGI_LINKS
 
 #if defined(EXEC_LINKS) || defined(EXEC_SCRIPTS)
 
