diff -ru lynx2-8-2.old/lynx.cfg lynx2-8-2/lynx.cfg
--- lynx2-8-2.old/lynx.cfg	Tue May 11 15:02:06 1999
+++ lynx2-8-2/lynx.cfg	Tue May 11 15:11:22 1999
@@ -43,7 +43,8 @@
 #
 # Normally we expect you will connect to a remote site, e.g., the Lynx starting
 # site:
-STARTFILE:http://lynx.browser.org/
+#STARTFILE:http://lynx.browser.org/
+STARTFILE:file://localhost/usr/share/doc/HTML/index.html
 #
 # As an alternative, you may want to use a local URL.  A good choice for this is
 # the user's home directory:
@@ -109,9 +109,9 @@
 # http://www.crl.com/~subir/lynx/lynx_help/lynx_help_main.html
 #   This should be changed to the local path.
 #
-HELPFILE:http://www.trill-home.com/lynx/lynx_help/lynx_help_main.html
+#HELPFILE:http://www.trill-home.com/lynx/lynx_help/lynx_help_main.html
 .ex
-#HELPFILE:file://localhost/PATH_TO/lynx_help/lynx_help_main.html
+HELPFILE:file://localhost/usr/share/lynx/help/lynx_help_main.html
 
 .h2 DEFAULT_INDEX_FILE
 # DEFAULT_INDEX_FILE is the default file retrieved when the
@@ -326,7 +327,7 @@
 # Older browser versions will fail so keep NCR_IN_BOOKMARKS:FALSE if you
 # plan to use them.
 #
-#NCR_IN_BOOKMARKS:FALSE
+NCR_IN_BOOKMARKS:TRUE
 
 # FORCE_8BIT_TOUPPER overrides locale settings and uses internal 8-bit
 # case-conversion mechanism for case-insensitive searches in non-ASCII display
@@ -1535,6 +1536,8 @@
 #VIEWER:image/x-tiff:xli %s&:XWINDOWS
 #VIEWER:image/jpeg:xli %s&:XWINDOWS
 #VIEWER:video/mpeg:mpeg_play %s &:XWINDOWS
+VIEWER:image/gif:zgv %s:NON_XWINDOWS
+VIEWER:image/jpeg:zgv %s:NON_XWINDOWS
 
 # The global and personal MAILCAP files allow you to specify external
 # viewers to be spawned when Lynx encounters different MIME types, which
@@ -1903,15 +1906,14 @@
 #
 # Uncomment and change any of the compilation defaults.
 #
-#COLOR:0:black:white
-#COLOR:1:blue:white
-#COLOR:2:yellow:blue
-#COLOR:3:green:white
-#COLOR:4:magenta:white
-#COLOR:5:blue:white
-#COLOR:6:red:white
-COLOR:6:brightred:black
-#COLOR:7:magenta:cyan
+COLOR:0:black:lightgray
+COLOR:1:blue:lightgray
+COLOR:2:yellow:blue
+COLOR:3:black:lightgray
+COLOR:4:black:lightgray
+COLOR:5:blue:lightgray
+COLOR:6:white:blue
+COLOR:7:lightgray:black
 
 # External application support.  This feature allows Lynx to pass a given
 # URL to an external program.  It was written for three reasons.
