diff -Nur lynx2-8-2/lynx.cfg lynx2-8-2.pld/lynx.cfg
--- lynx2-8-2/lynx.cfg	Fri Jun  4 14:31:38 1999
+++ lynx2-8-2.pld/lynx.cfg	Fri Jun  4 14:31:10 1999
@@ -1,12 +1,12 @@
 # lynx.cfg file.
-# The default placement for this file is /usr/local/lib/lynx.cfg (Unix)
+# The default placement for this file is /usr/share/lynx/lynx.cfg (Unix)
 #                                     or Lynx_Dir:lynx.cfg (VMS)
 #
 #
 # Definition pairs are of the form  VARIABLE:DEFINITION
 # NO spaces are allowed between the pair items.
 #
-#  If you do not have write access to /usr/local/bin you may change
+#  If you do not have write access to /usr/bin you may change
 #  the default location of this file in the userdefs.h file and recompile,
 #  or specify its location on the command line with the "-cfg"
 #  command line option.
@@ -32,7 +32,7 @@
 #
 # Then in ~/lynx.cfg:
 #
-#   INCLUDE:/usr/local/lib/lynx.cfg
+#   INCLUDE:/usr/share/lynx/lynx.cfg
 #           ^^^^^^^^^^^^^^^^^^^^^^^ or whatever is appropriate on your system
 # and now your own tweaks.
 #
